source/codegen/validate_source_protos.py

```diff
@@ -0,0 +1,54 @@
+"""Script for validating the source proto files against generated ones.
+
+Expects input source path to source/codegen/protos and generated folder path.
+"""
+
+import argparse
```

Comment:
Plug to consider `click` ([Internal best practices doc](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/21764/Common-Modules-List)) when you can use external modules. (No need for a rewrite here; and always optional IMO).

Reply:
Oh neat. I went ahead and did it. I like `click`, don't think I've seen it before.